src/test/java/com/celements/cleverreach/util/CssInlinerTest.java

```diff
@@ -38,13 +39,13 @@ public void testInline_noStyles() {
     try {
       cssInliner.inline("<div></div>", (String) null);
       fail("Expecting NPE");
-    } catch (NullPointerException npe) {
+    } catch (CssInlineException | NullPointerException npe) {
```

Comment:
What is expected? A CssInlineException or a NPE? The fail-message suggests that an NPE is expected. The Catch-Statement suggests that, either could happen AND is as expected in the given use case. Can you please fix the Fail-Message or the catch statement?